.circleci/config.yml

```diff
@@ -22,6 +22,8 @@ workflows:
           requires: [orb-tools/lint]
       - orb-tools/publish-dev:
           orb-name: deepsource/test-coverage
+          publish-sha-version: false
```

Comment:
The sha version is used for the final publishing step.

Reply:
actually, this line can just be removed.

Reply:
Sure. I was using `orb-tools@9.0.0` in the previous version and noticed that the sha-version was being published instead of `dev:alpha`.

I'll make the changes here. Waiting for the complete review.